src/main/webapp/listings.html

```diff
@@ -6,16 +6,24 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload="fetch_auth_info();">
+  <body onload="loadListings();">
     <ul>
         <li><a href="index.html">Home</a></li>
         <li><a href="listings.html">View Listings</a></li>
         <li><a href="new-listing.html">Create A Post</li>
         <li><a href="#" id="loglink"></a></li>
-    </ul> 
-    <h1>Listings!</h1>
-    <div id="content">
-      <div id="commentCont"></div>
+    </ul>
+    <div class="content-container">
+      <h1>Listings!</h1>
+      <div id="listings-container"></div>
+      <div class="listing" id="listing-template" hidden>
```

Comment:
After your PR and our conversation I discovered that HTML supports this scenario with templates (https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_templates_and_slots, https://www.w3schools.com/tags/tag_template.asp). Feel free to keep the implementation as is.

Reply:
I see. Seems like it's almost identical except the wrapping template tag. I'd like to discuss this during our 1-1 if that's cool. I'll keep this implementation for the sake of getting the MVP out, and if I have time I'll improve it.